### ISSUE TITLE
North Dakota cases shows daily diffs NOT totals

### ISSUE THREAD

**Issue by CovidCountyUSA** [covid-projections/covid-data-public#164]
North Dakota cases field seems to show daily differences instead of totals for all counties and all dates.

**Comment by mikelehen** [comment]
@CovidCountyUSA Can you clarify what you are looking at exactly?  Is this an API endpoint? Which field/column are you looking at?

**Comment by CovidCountyUSA** [comment]
I'm looking at the https://github.com/covid-projections/covid-data-public/blob/master/data/cases-covid-county-data/timeseries-common.csv file.

Column: "cases"
state = ND
counties = seems to be all counties in ND
date range: all dates

Issue:
The "cases" column for counties in North Dakota for All Dates.  The "cases" column seems to reflect the daily differences as opposed to the  accumulated total number of cases.  I only checked the "cases" column. I haven't checked the other columns.  All the counties in all the other states seem to show accumulated cases. 

BTW Thanks for putting the data together

**Comment by mikelehen** [comment]
@CovidCountyUSA Oh, you're right. Thanks for the report!

That file is actually data that we import from one of our data sources (https://covidcountydata.org/).  I have forwarded the bug report to them so they can investigate.

I will point out that our website (https://covidactnow.org/) doesn't actually use the case data in that file.  Instead, we use case data from New York Times (https://github.com/nytimes/covid-19-data).

Also, this repository is part of our daily data pipeline and is open for people to look at, but we don't really intend for anybody to be consuming data from this repository directly.  Files could be removed or renamed, etc. as we make changes to our pipeline.  Instead, we have a public API which you can check out here: https://apidocs.covidactnow.org/. If you intend to consume any of our data, that would definitely be the best way to do it.  

Thanks!

**Comment by CovidCountyUSA** [comment]
Glad to be of help.  I need a csv file because I'm doing my own parsing/analysis and csv is what I geared up for.  I like the/your csv in question because it's pretty clean and consistent.

**Comment by mikelehen** [comment]
@CovidCountyUSA Our API includes CSVs which will hopefully meet your needs!  Take a look at https://apidocs.covidactnow.org/access#historical-data-for-all-counties-csv. 

If there's something missing / lacking, feel free to holler.

**Comment by CovidCountyUSA** [comment]
Thanks for your help.  Here's what I did with the data
https://youtu.be/meswiMV8q1A

**Comment by mikelehen** [comment]
Hah, very nice!  We actually did something similar, though it's a bit less dramatic without the soundtrack. 😁  https://blog.covidactnow.org/covid-spread-us-analysis/

Anyway, nice work! I shared this with the rest of the team. I think they'll get a kick out of it.

**Comment by CovidCountyUSA** [comment]
Thanks for the feedback! The map graphic generated, for lack of a better
term, "in real time"..meaning it's not a rendering but a real time graphing
of the data on a javascript based frontend that I custom wrote.   That
video was just a mild preview of what I hope will be a fully interactive,
multi-algo website interface.

I didn't see the video your team did before.  That's the sort of thing I
started out generating but I found that "non-data" people didn't respond
well to the noisy graphics, so I worked on several smoothing algorithms
till I found a few that produced the cloud-like graphic that represents the
spread more accurately on macro level.  It lends itself to graphically
showing the "motion" of the spread, which one tends to see in one's mind
when they think of  a "spreading virus"  .   When I showed the graphic with
the algo you saw, I got some gasps...which is what I was shooting for 😎.
